Makefile

```diff
@@ -16,18 +16,10 @@ image-build:
 	docker build \
 		--build-arg GOLANG_VERSION=$(GOLANG_VERSION) \
 		--build-arg GOBORING_BUILD=$(GOBORING_BUILD) \
-		--tag $(ORG)/hardened-build-base:$(TAG) \
 		--tag $(ORG)/hardened-build-base:$(TAG)-$(ARCH) \
-		.
+		. \
+		-f Dockerfile.$(ARCH)
```

Comment:
Let us please continue to build the arch-less tag. Combined with the `ORG` envvar in the `Makefile` this is handy for "local" development workflows.
```makefile
	docker build \
		--build-arg GOLANG_VERSION=$(GOLANG_VERSION) \
		--build-arg GOBORING_BUILD=$(GOBORING_BUILD) \
		--tag $(ORG)/hardened-build-base:$(TAG) \
		--tag $(ORG)/hardened-build-base:$(TAG)-$(ARCH) \
		--file Dockerfile.$(ARCH)
		.
```